README.md

```diff
@@ -1,178 +1,1234 @@
-# AWS Cloud Development Kit (AWS CDK)
+# AWS CodePipeline Actions
```

Comment:
This seems like an unintentional change. Our root level README shouldn't be edited on this PR.

Reply:
Hi again!

I have undone the changes.

Reply:
Hi again Kendra.. Yea it has confused me how that went in but I have rolled back the changes now.